subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/AbstractConsoleBuildPhaseFunctionalTest.groovy

```diff
@@ -304,6 +307,8 @@ abstract class AbstractConsoleBuildPhaseFunctionalTest extends AbstractIntegrati
             include 'lib'
             include 'util'
         """
+
+        enableIncrementalArtifactTransformations(settingsFile)
```

Comment:
Depending on what kind of execution we choose, a different number of console logs is produced, since the project workspaces are different. I didn't want to test both, since the non-incremental execution will go away. So I opted for enabling incremental transformations in this test.